src/main/learnocaml_server_main.ml

```diff
@@ -48,52 +53,68 @@ let main o =
   in
   exit (run ())
 
-let man = [
-  `S "DESCRIPTION";
+let man = 
+  let open Manpage in
+  [
+  `S s_description;
   `P "This is the server for learn-ocaml. It is equivalent to running \
       $(b,learn-ocaml serve), but may be faster if compiled to native code. It \
       requires the learn-ocaml app to have been built using $(b,learn-ocaml \
       build) beforehand.";
-  `S "SERVER OPTIONS";
-  `S "AUTHORS";
-  `P "Learn OCaml is written by OCamlPro. Its main authors are Benjamin Canou, \
-      Çağdaş Bozman, Grégoire Henry and Louis Gesbert. It is licensed under \
-      the MIT License.";
-  `S "BUGS";
+  `S s_options;
+  `S s_authors;
+  `P "The original authors are Benjamin Canou, \
+      Çağdaş Bozman, Grégoire Henry and Louis Gesbert (OCamlPro). \
+      Learn-OCaml is licensed under the MIT License.";
+  `S s_bugs;
   `P "Bugs should be reported to \
       $(i,https://github.com/ocaml-sf/learn-ocaml/issues)";
 ]
 
 let app_dir =
-  let open Cmdliner.Arg in
+  let open Arg in
   value & opt string "./www" & info ["app-dir"; "o"] ~docv:"DIR" ~doc:
     "Directory where the app has been generated by the $(b,learn-ocaml build) \
      command, and from where it will be served."
 
 let base_url =
-  let open Cmdliner.Arg in
+  let open Arg in
   value & opt string "" &
-    info ["base-url"] ~docv:"BASE_URL" ~env:(env_var "LEARNOCAML_BASE_URL") ~doc:
+    info ["base-url"] ~docv:"BASE_URL" ~env:(Cmd.Env.info "LEARNOCAML_BASE_URL") ~doc:
       "Set the base URL of the website. \
        Should not end with a trailing slash. \
        Currently, this has no effect on the backend - '$(b,learn-ocaml serve)'. \
        Mandatory for '$(b,learn-ocaml build)' if the site is not hosted in path '/', \
        which typically occurs for static deployment."
 
-let main_cmd =
-  Cmdliner.Term.(const main $ Learnocaml_server_args.term app_dir base_url),
-  Cmdliner.Term.info
+let  exits =
+  let open Cmd.Exit in
+  [ info ~doc:"Default exit." ok
+  ; info ~doc:"Uncaught exception." 2
+  ; info ~doc:"Server error whose cause is printed on stderr." 10
+  ; info ~doc:"Aborting. Server internal error occured in less then 15 seconds after the launch." 20
+  ; info ~doc:"Termination signal (SIGTERM) recieved by server." 113
+  ; info ~doc:"Interruption signal (SIGINT) recieved by server." 126
```

Comment:
Hi @hernoufM, sorry for replying late.

I reviewed the PR anew, pushed a tiny commit with a few tweaks, and everything looks very fine!
apart from these two lines:
are you sure about the need to include these two lines?

because I believe every time there is a signal, the exit code should be > 128.

And if I do this example session:

```
$ learn-ocaml build --repo=demo-repository
Learnocaml v.0.14.0 running.
Updating app at ./www
demo                     (no changes)
demo2                    (no changes)
$ learn-ocaml-server
Learnocaml server v.0.14.0 starting on port 8080
Found the following teacher tokens:
  - X-HZS-TUJ-XOW-VRK
^CSIGINT caught: stopping
$ echo $?
134

$ learn-ocaml-server
Learnocaml server v.0.14.0 starting on port 8080
Found the following teacher tokens:
  - X-HZS-TUJ-XOW-VRK
Killed
$ echo $?
137
```

and AFAIK these exit codes should be standard, so I'd say, no need to document them.

So I propose to push another small commit to remove these two lines and merge afterwards. WDYT?

Reply:
> So I propose to push another small commit to remove these two lines (…)

Done just now, FYI.